packages/plugin-locale/src/index.ts

```diff
@@ -194,7 +194,11 @@ export default (api: IApi) => {
         Antd: !!antd,
         LocaleList: localeList,
         ShowSelectLang: localeList.length > 1 && !!antd,
-        iconsPkgPath: winPath(require.resolve('@ant-design/icons')),
+        iconsPkgPath: winPath(
+          require.resolve(
+            `@ant-design/icons${api.config?.ssr ? '' : '/es/icons'}`,
+          ),
```

Comment:
```suggestion
          dirname(require.resolve('@ant-design/icons/package.json')),
```

将何种 module 引用决定权将给 webpack

Reply:
done